test/libsolidity/syntaxTests/controlFlow/uninitializedAccess/unreachable.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+    uint[] s;
+    function f() internal returns (uint[] storage a)
+    {
+        revert();
+        a[0] = 0;
```

Comment:
How difficult is it to warn about unreachable code here?

Reply:
We'd have to think about the conditional case (in ``c = false ? a : b;`` is ``a`` unreachable or is "unreachable code" restricted to entire statements?), resp. control flow breaks within statements, etc.. So non-trivial, but should be doable. But I'd separate it from this PR.